templates/internet-of-things/appstore.html

```diff
@@ -0,0 +1,220 @@
+{% extends "internet-of-things/base_things.html" %}
+{% block title %}White label app store for IOT | Ubuntu for the Internet of Things {% endblock %}
+{% block meta_description %}For cloud based and enterprise IoT app stores, control, manage and monetise software deployments on your connected Linux devices.{% endblock meta_description %}
+
+{% block content %}
+
+<section class="p-strip--accent is-deep">
+  <div class="row u-equal-height">
+    <div class="col-8">
+      <h1>An IoT app store for all your devices</h1>
+      <p>Manufacturers can cut the cost of managing software updates on their devices and take advantage of new revenue opportunities by selling services via their own branded IoT app store &mdash; hosted by Canonical.</p>
+      <ul class="p-inline-list">
+        <li class="p-inline-list__item">
+          <a class="p-button--positive" href="https://www.brighttalk.com/webcast/6793/290087" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Webinar', 'eventLabel' : 'Watch the Path To IoT Revenue webinar', 'eventValue' : undefined });"><span class="p-link--external">Watch the 'Path To IoT Revenue' webinar</span></a>
+        </li>
+        <li class="p-inline-list__item">
+          <a href="/internet-of-things/contact-us?product=appstore" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Contact us', 'eventLabel' : 'Telcos - Top CTA' });" class="p-button--neutral">Contact us&nbsp;&rsaquo;</a>
+        </li>
+      </ul>
+    </div>
+    <div class="col-4 u-align--center u-vertically-center u-hide--small">
+      <img class="hero-image" src="{{ ASSET_SERVER_URL }}1570841c-icon-app-store.svg" alt="">
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-deep is-bordered">
+  <div class="row p-divider">
+    <div class="col-4 p-divider__block">
+      <h2 class="p-heading-three">Control the user&rsquo;s experience</h2>
+      <p>The option of a private or public store enables you to select which software and which apps are available to your customers, while device access ensures only authenticated users can access your store.</p>
```

Comment:
Note: this is a long sentence without breaks.

Reply:
Fixed